### PR TITLE
make consisent cart design for all products

### LINKED ISSUE #3673: Make cart design consistent for all types of products
## What is the motivation for adding / enhancing this feature?
After introducing possibility to edit product properties (color, size) from cart level, product box for them was redesigned - quantity field, remove button etc. were moved to new positions. 

However, for products that don't have such options, old design is still in use. So in case you have both types of products in cart at the same time, layout of fields and buttons looks inconsistent and might be confusing.  

design

## What are the acceptance criteria 
Layout of productbox in cart (inlcuding quantity fields, remove buttons etc. is consistent for all types of products. It might be a good idea to ask @StasiekDivante to prepare a design? :)

## Can you complete this feature request by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [x] This is a normal feature request. This should be available on https://test.storefrontcloud.io and then after tests this can be added to next Vue Storefront version. In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] (**Pick this option only if you're sure**) This is an important improvement request for current Release Candidate version on https://next.storefrontcloud.io and should be placed in next RC version. In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] (**Pick this option only if you're sure**) This is a critical improvement request for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version. In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

### PR DESCRIPTION
**Pull request by gibkigonzo**
### Related issues

closes #3673

### Short description and why it's useful

This change removes the conditions for adding classes for non-configurable products.

### Screenshots of visual changes before/after (if there are any)

After:
Zrzut ekranu z 2019-10-02 08-21-03

### Which environment this relates to
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [x] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [x] I've updated the [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/themes/default/components/core/blocks/Microcart/Product.vue [hunk 1]
```diff
@@ -18,38 +18,36 @@
...
+              :to="productLink"
+              data-testid="productLink"
+              @click.native="$store.commit('ui/setMicrocart', false)"
+            >
+              {{ product.name | htmlDecode }}
+            </router-link>
+            <div class="h6 cl-bg-tertiary pt5 sku" data-testid="productSku">
+              {{ product.sku }}
+            </div>
+            <div class="h6 cl-bg-tertiary pt5 options" v-if="isOnline && !editMode && product.totals && product.totals.options">
```
Comment (andrzejewsky) [inline src/themes/default/components/core/blocks/Microcart/Product.vue]: please move this condition (v-if)to the computed value eg. `isTotalsActive`

File: src/themes/default/components/core/blocks/Microcart/Product.vue [hunk 2]
```diff
@@ -18,38 +18,36 @@
...
-              <div class="h6 pt5 cl-success" v-if="product.info && Object.keys(product.info).length > 0 && Object.keys(product.errors).length === 0">
-                {{ product.info | formatProductMessages }}
+            </div>
+            <div class="h6 cl-bg-tertiary pt5 options" v-else-if="!editMode && product.options">
+              <div v-for="opt in product.options" :key="opt.label">
+                <span class="opn">{{ opt.label }}: </span>
+                <span class="opv" v-html="opt.value" />
               </div>
             </div>
+            <div class="h6 pt5 cl-error" v-if="product.errors && Object.keys(product.errors).length > 0">
```
Comment (andrzejewsky) [inline src/themes/default/components/core/blocks/Microcart/Product.vue]: same: v-if to computed eg. `hasProductErrors`

File: src/themes/default/components/core/blocks/Microcart/Product.vue [hunk 3]
```diff
@@ -18,38 +18,36 @@
...
+              {{ product.errors | formatProductMessages }}
+            </div>
+            <div class="h6 pt5 cl-success" v-if="product.info && Object.keys(product.info).length > 0 && Object.keys(product.errors).length === 0">
```
Comment (andrzejewsky) [inline src/themes/default/components/core/blocks/Microcart/Product.vue]: create computed `hasProductInfo` and combine two of them `hasProductInfo && !hasProductErrors`

File: src/themes/default/components/core/blocks/Microcart/Product.vue (hunk 1 above)
Comment (gibkigonzo) [reply src/themes/default/components/core/blocks/Microcart/Product.vue]: done

File: src/themes/default/components/core/blocks/Microcart/Product.vue (hunk 2 above)
Comment (gibkigonzo) [reply src/themes/default/components/core/blocks/Microcart/Product.vue]: done

File: src/themes/default/components/core/blocks/Microcart/Product.vue (hunk 3 above)
Comment (gibkigonzo) [reply src/themes/default/components/core/blocks/Microcart/Product.vue]: and done :)

**Pull request merged**